content/200-concepts/050-overview/100-what-is-prisma/index.mdx

```diff
@@ -19,7 +19,7 @@ To learn more about the _motivation_ for Prisma, check out the [**Why Prisma?**]
 Prisma is an [open source](https://github.com/prisma/prisma) next-generation ORM. It consists of the following parts:
 
 - **Prisma Client**: Auto-generated and type-safe query builder for Node.js & TypeScript
-- **Prisma Migrate** (_experimental_): Declarative data modeling & migration system
+- **Prisma Migrate** (Preview): Declarative migration system
```

Comment:
I'd remove "Declarative" here, as the system is using both declarative and imperative approaches in combination.
```suggestion
- **Prisma Migrate** (Preview): Migration system
```